sklearn/decomposition/_dict_learning.py

```diff
@@ -1378,6 +1408,21 @@ class MiniBatchDictionaryLearning(SparseCodingMixin, BaseEstimator):
         RandomState instance that is generated either from a seed, the random
         number generattor or by `np.random`.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.datasets import make_sparse_coded_signal
+    >>> from sklearn.decomposition import MiniBatchDictionaryLearning
+    >>>
+    >>> np.random.seed(42)
+    >>> X, dictionary, code = make_sparse_coded_signal(
+    ... n_samples=100, n_components=15, n_features=20, n_nonzero_coefs=10)
```

Comment:
Maybe emphasize that `X` is dense originally: `np.mean(X == 0)` should be `0.0`.

Reply:
Done this

Reply:
The pull request interface on github does not show the change. Have you pushed the newest commits in your branch to the matching branch in your github fork?

Reply:
Please check now